src/crypto/crypto_cipher.cc

```diff
@@ -901,6 +901,14 @@ bool CipherBase::Final(std::unique_ptr<BackingStore>* out) {
   if (kind_ == kDecipher && IsSupportedAuthenticatedMode(ctx_.get()))
     MaybePassAuthTagToOpenSSL();
 
+  // OpenSSL v1.x doesn't verify the presence of the auth tag so do
```

Comment:
Should we add a comment in here to remove this when v16 is not supported?

Reply:
I believe this comment already serves that purpose. It mentions v1.x which when it's time for a codebase cleanup will get picked up?

Reply:
Once we fully drop 1.1.1 (including through dynamic linking), we'll have to revisit all version check macros etc. anyway.